src/Serialization/Fields.jl

```diff
@@ -268,6 +268,79 @@ function load_internal_with_parent(s::DeserializerState,
     return parent(num, den)
 end
 
+################################################################################
+# RationalFunctionField
+
+@registerSerializationType(AbstractAlgebra.Generic.RationalFunctionField)
```

Comment:
Note that this is different to what your code did before; that would be equivalent to 
```suggestion
@registerSerializationType(AbstractAlgebra.Generic.RationalFunctionField, "RationalFunctionField")
```
Just wanted to point this out to make sure it is intentional; either is fine for me at this point (someday we need to discuss how we name these things if we truly want this to become a format others can read, but not now)

Reply:
ah, sorry, it wasn't intentional, thanks for catching this, i will change this for Rat as well